thinkshout/bene
# #205 - remove patch definitions in composer.json that have been commi…

see comments on https://github.com/thinkshout/bene/issues/205 for verification patches exist in stable versions

to test:

1. clone https://github.com/thinkshout/bene-project/tree/4-fix-install
2. run `./scripts/install.sh -d [some-dir]`
3. observe that installation completes with no patch warnings (would happen before "setup git" user input step)
